### PR TITLE
Update zh-CN.json

### PR DESCRIPTION
**Pull request by millet6**
更新汉化内容
Update Chinese content

## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Type of changes

- [ ] Bug fix
- [ ] New feature 
- [ ] Enhancement
- [x] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

File: src/main/resources/languages/zh-CN.json [hunk 1]
```diff
@@ -28,8 +28,8 @@
         "login_token_attempt": "[Dispatch] 客户端 %s 正在尝试通过 token 登录",
         "login_token_error": "[Dispatch] 客户端 %s 通过 token 登录失败",
         "login_token_success": "[Dispatch] 客户端 %s 已通过 token 登录，UID 为 %s",
-        "login_password_error": "🇺🇸[Dispatch] Client %s failed to log in via password.",
-        "login_password_storage_error": "🇺🇸[Dispatch] Client %s failed to log in via password because there is no password in the database.",
+        "login_password_error": "🇺🇸[Dispatch] 客户端 %s 无法通过密码登录",
+        "login_password_storage_error": "🇺🇸[Dispatch] 客户端 %s 无法登录！ 因为数库中没有密码",
```
Comment (Birdulon) [inline src/main/resources/languages/zh-CN.json]:
```suggestion
        "login_password_error": "[Dispatch] 客户端 %s 无法通过密码登录",
        "login_password_storage_error": "[Dispatch] 客户端 %s 无法登录！ 因为数库中没有密码",
```
Remove these when you update the strings :)

File: src/main/resources/languages/zh-CN.json [hunk 2]
```diff
@@ -257,12 +257,12 @@
...
-      "range_error": "Constellation level must be between 0 and 6.",
-      "level_error": "Invalid constellation level.",
-      "fail": "Failed to set constellation.",
-      "failed_success": "Constellations for %s have been set to %s. Please reload scene to see changes.",
-      "success": "Constellations for %s have been set to %s.",
-      "description": "Sets constellation level for your current active character"
+      "range_error": "命座等级必须是 0~6级",
+      "level_error": "无效的命座等级",
+      "fail": "命座等级设置失败",
+      "failed_success": "命座 %s 已设置为 %s. 请传送到[渊下宫]或重新登录查看效果",
```
Comment (SleepyAsh0191) [inline src/main/resources/languages/zh-CN.json]: 原文并没有提到渊下宫，实际上是重载场景。
The original text not mentioned Enkanomiya.

File: src/main/resources/languages/zh-CN.json (hunk 1 above)
Comment (SleepyAsh0191) [inline src/main/resources/languages/zh-CN.json]: 错别字：数库 -> 数据库
Typo: datase -> database

**Review by KingRainbow44 — APPROVED** [review]
Approved on behalf of @Kimi898246

**Pull request merged**